website/src/_posts/2021-12-2.1-2.3.md

```diff
@@ -10,17 +10,17 @@ image: https://uppy.io/images/blog/2.1-2.3/audio-cover.jpg
 published: true
 ---
 
-Last Christmas we gave you [Uppy 1.24](https://uppy.io/blog/2020/12/1.24/), this year we’ll take it away (since it’s outdated by now) and give you brand new Uppy 2.3 🎁!
+Last Christmas, we gave you [Uppy 1.24](https://uppy.io/blog/2020/12/1.24/), but this very next year, we’ll take it away (since it’s outdated by now) and give you a brand-new Uppy 2.3 🎁!
```

Comment:
This line doesn't pass the linter validation.

Reply:
Why could that be?

I'm not exactly sure what linter validation is, but I don't think I made erroneous edits here.

Reply:
Our _very_ picky linter thinks we shouldn't use `very` in our docs – I suppose our tooling doesn't know about Mariah Carey 🙄😅
This fix is to add a `<!--retext-simplify ignore very-->` comment before the sentence to silence the error.